api/v1alpha1/mutation_types.go

```diff
@@ -50,12 +50,22 @@ type MutationSpec struct {
 type ValuesSource struct {
 	// +optional
 	FluxSource *FluxValuesSource `json:"fluxSource,omitempty"`
-
-	// TODO
-	// ConfigMap meta.LocalObjectReference
+	// +optional
+	ConfigMapSource *ConfigMapSource `json:"configMapSource,omitempty"`
+	// +optional
+	Key string `json:"key,omitempty"`
```

Comment:
Wasn't this key supposed to be the key inside the ConfigMapSource? Maybe it's a remnant of an older implementation?